corehq/apps/api/odata/utils.py

```diff
@@ -23,3 +24,24 @@ def get_case_type_to_properties(domain):
             cleaned_case_property = export_group_schema.label.replace('_', '')
             case_type_to_properties[case_type].append(cleaned_case_property)
     return dict(case_type_to_properties)
+
+
+def get_xmlns_by_app(domain, app_id):
+    app = get_current_app(domain, app_id)
+    return [form.xmlns.split('/')[-1] for form in app.get_forms()]
```

Comment:
if you're just returning the unique bit at the end it might be worth using a more accurate name for the function. I'd expect this one to return the full `xmlns`